frontend/src/main/components/Commons/CommonsOverview.js

```diff
@@ -15,6 +16,17 @@ export default function CommonsOverview({ commonsPlus, currentUser }) {
             <Card.Header as="h5">Announcements</Card.Header>
             <Card.Body>
                 <Row>
```

Comment:
This should probably be a for each that iterates over *all* of the announcements for the commons.

As an example, see: https://github.com/ucsb-cs156/proj-happycows/blob/e7487fca555481a59c9ab47d9583b3d2cc8f4a01/frontend/src/main/components/Commons/CommonsList.js#L38

Reply:
Thank you, professor! My teammate has helped me fix this. Now in PlayPage.js, all announcements will be extracted instead of the latest one. Now the play page can show all announcements. However, we find that it is too tricky to pass the mutation test if we add the function to optionally hide the announcement. So, we follow the TA's suggestion to remove the "show more/less" button. Now every announcement is just scrollable if it is too large